config/rbac/mutation/assignmetadata_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit assignmetadatas.
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
None of these roles are necessary, they should all be folded up into the gatekeeper-manager role

Reply:
Done